src/compiler/types.ts

```diff
@@ -1502,6 +1503,7 @@ namespace ts {
     export type LogicalOperator

```

Comment:
This is actually kind of strange because it's not *really* a LogicalOperator. Should it be moved out?